src/main/typescript/components/form/Validation.tsx

```diff
@@ -362,6 +366,28 @@ export function validateSpatialBoxes(spatialCoordinateSettings: SpatialCoordinat
     })
 }
 
+export function validateFiles(files: Files): { [filepath in keyof Files]: ReactElement | string | undefined } | undefined {
```

Comment:
Is this validation done at submit? I vaguely remember a low level configuration value to maximize upload sizes. It seems better to me to prevent storing too large files at all.